src/app.html

```diff
@@ -29,16 +57,19 @@ <h1 class="header center -text text-lighten-2 thin">
                         <%= htmlWebpackPlugin.options.description  %>
                     </p>
                 </div>
+                <div class="row show-on-large center">
+                    <a class="call-to-action" href="#resume"><i class="fa fa-download fa-fw"></i> View My Resume and Contact Info</a>
+                </div>
                 <div class="row hide-on-large-only center">
-                    <a class="white-text obfuscate btn-flat btn-large" href="mailto:archie...trajano...net"><i class=" material-icons left">email</i> archie...trajano...net</a>
+                    <a class="white-text obfuscate btn-flat" href="mailto:archie...trajano...net"><i class=" material-icons left">email</i> archie...trajano...net</a>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)